hyperframe/frame.py

```diff
@@ -30,7 +30,7 @@ class Frame(object):
     # it must be zero. If 'either', it's not checked.
     stream_association = None
 
-    def __init__(self, stream_id):
+    def __init__(self, stream_id=0):
```

Comment:
I think doing this for the base class runs the risk of hiding bugs in implementations of new frame types. I'm happy for the leaf frame classes to do this, I'm less happy about the base class doing it.